app/src/main/java/org/mozilla/tv/firefox/channels/ChannelConfig.kt

```diff
@@ -16,19 +18,26 @@ data class ChannelConfig(
     val onClickTelemetry: ((ChannelTile) -> Unit)? = null,
     val onLongClickTelemetry: ((ChannelTile) -> Unit)? = null,
     val onFocusTelemetry: ((ChannelTile, Boolean) -> Unit)? = null,
-    val itemsMayBeRemoved: Boolean = false
+    val itemsMayBeRemoved: Boolean = false,
+    val isEnabledInCurrentExperiment: Boolean,
+    val enabledInLocales: KillswitchLocales
 ) {
     companion object {
         fun getPocketConfig(): ChannelConfig = ChannelConfig(
                 onClickTelemetry = { tile -> TELEMETRY.pocketVideoClickEvent(tile.id) },
                 // TODO focus telemetry should probably only be sent on focus gain, but this is
                 //  how our previous implementation worked. Keeping this to maintain data consistency
-                onFocusTelemetry = { tile, _ -> TELEMETRY.pocketVideoImpressionEvent(tile.id) }
+                onFocusTelemetry = { tile, _ -> TELEMETRY.pocketVideoImpressionEvent(tile.id) },
+                isEnabledInCurrentExperiment = true,
+                // Pocket is enabled in all countries, for any English locale
+                enabledInLocales = KillswitchLocales.ActiveIn(Locale("en", ""))
```

Comment:
I wonder if we should send telemetry describing which features are available to users rather than the data team having to piece it together (and thus duplicate logic) from our "which experiments is the user in" and "which locale is the user in" telemetry. Maybe file a bug to investigate?